chemprop/v2/models/multi.py

```diff
@@ -16,24 +16,22 @@ def __init__(
         self,
         message_passing: MulticomponentMessagePassing,
         agg: Aggregation,
-        ffn: nn.Sequential,
-        transform: OutputTransform,
-        loss_fn: LossFunction,
-        metrics: Iterable[Metric],
-        task_weights: Tensor | None = None,
+        readout: Readout,
+        batch_norm: bool = True,
+        metrics: Iterable[Metric] | None = None,
+        w_t: Tensor | None = None,
         warmup_epochs: int = 2,
         init_lr: float = 1e-4,
         max_lr: float = 1e-3,
         final_lr: float = 1e-4,
     ):
         super().__init__(
-            message_passing,
+            message_passing, # MPNN doesn't take MulticomponentMessagePassing?
```

Comment:
the two have largely the same interface, so we can rely on the superclass methods to take care of a bunch of boilerplate even if the objects are not the same